thirdeye/thirdeye-pinot/src/main/resources/schema/create-schema.sql

```diff
@@ -1,3 +1,5 @@
+create alias if not exists TO_UNIXTIME as $$ long unix_timestamp(java.sql.Timestamp timestamp) { return
```

Comment:
Is this still required? If so, can you mention where and how?

Reply:
Yes, it is required by SqlUtils for H2 database to convert timestamp to unix epoch time.